Add `uniquename` property to application resource (#9532)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/application.md

| tokenEncryptionKeyId       | String                                                         | Specifies the keyId of a public key from the keyCredentials collection. When configured, Azure AD encrypts all the tokens it emits by using the key this property points to. The application code that receives the encrypted token must use the matching private key to decrypt the token before it can be used for the signed-in user.                                                                                                                                                                                                                     |

> | uniqueName | String | The unique identifier for the application that's assigned to it when it's created through Azure Resource Manager. Immutable. Read-only. |

Reviewer: ```suggestion
| uniqueName | String | The unique identifier that's assigned to an application when the application is created through Azure Resource Manager. Immutable. Read-only. |
```
